### PR TITLE
cp URL fix 1

### PR DESCRIPTION
**Pull request by stephamon**
first set of fixes

JIRA 2694

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-d06d56b576/how-to/). :bow:

File: content/exchange/cloud-office-control-panel-overview.md [hunk 1]
```diff
@@ -160,4 +160,4 @@ The following options available in the **Support** menu:
 -  **Email Setup Wizard** - View useful mailbox information while using step-by-step instructions on how to set up your Cloud Office services.
 -  **View Tickets** - Create a new ticket and view any past tickets on the Support Tickets landing page.
 
-If you need assistance help a specific feature, read about how to use the [Help tool for hosted email and Skype for business](https://support.rackspace.com/how-to/help-tool-for-hosted-email-and-skype-for-business/).
+If you need assistance help a specific feature, read about how to use the [Help tool for hosted email and Skype for business](/how-to/help-tool-for-hosted-email-and-skype-for-business).
```
Comment (RackerKate) [inline content/exchange/cloud-office-control-panel-overview.md]: I think this should be "If you need help with a specific feature..."

File: content/general/change-account-information.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Request the **Service Transfer** form from your account manager.
 
 ### All other customers
 
-All other customers can find links to the forms described in this article by logging in to the [Cloud Control Panel](http://mycloud.rackspace.com) and selecting **Account** > **Docs and Forms**.
+All other customers can find links to the forms described in this article by logging in to the [Cloud Control Panel](http://login.rackspace.com) and selecting **Account > Docs and Forms**.
```
Comment (RackerKate) [inline content/general/change-account-information.md]: Change to "https."

File: content/general/change-account-information.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ In the **Cloud Account Number** and the **New Cloud Account Name** columns, prov
 
 ### Best practices for legal change forms
 
-The Service Transfer and Primary Contact Change forms have many required fields that must be complete before the Billing team can process the forms. After you have completed the form, scan the document and attach it to the support ticket in the [Cloud Control Panel](http://mycloud.rackspace.com) or the [MyRackspace Control Panel](https://my.rackspace.com).
+The Service Transfer and Primary Contact Change forms have many required fields that must be complete before the Billing team can process the forms. After you have completed the form, scan the document and attach it to the support ticket in the [Cloud Control Panel](http://login.rackspace.com) or the [MyRackspace Control Panel](https://login.rackspace.com).
```
Comment (RackerKate) [inline content/general/change-account-information.md]: change to the first link to use "https."

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-cced4056c3/how-to/). :bow:

File: content/exchange/add-an-activesync-or-bes-license.md [hunk 4]
```diff
@@ -26,9 +26,9 @@ this article to add an ActiveSync or BES license.
 
    - US Cloud Office customers, use <https://cp.rackspace.com/>.
 
-   - International and MyRackspace Cloud Office customers, use
-     <https://my.rackspace.com/> and navigate to **Products** &gt;
-     **Cloud Office** &gt; **Open Mail Control Panel**.
+   - International and MyRackspace Cloud Office customers, log in to
+     <https://login.rackspace.com/>. In the top navigation bar click **Select a Product > Dedicated Hosting**. Then, in the top navigation bar, click **Products >
```
Comment (RackerKate) [inline content/exchange/add-an-activesync-or-bes-license.md]: Suggest "In the top navigation bar, click..., then, click **Products > Cloud Office > Open Mail Control Panel**".

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a7eb9f3e7e/how-to/)

**Pull request merged**